src/main.js

```diff
@@ -8,6 +8,15 @@ import 'bootstrap/dist/css/bootstrap.css';
 import 'bootstrap-vue/dist/bootstrap-vue.css';
 Vue.use(BootstrapVue);
 
+// Load and setup vue-cookies
+import VueCookies from 'vue-cookies'
```

Comment:
You should probably use localstorage instead.

Reply:
Fixed